.devcontainer/devcontainer.json

```diff
@@ -1,5 +1,4 @@
 {
-  "$schema": "https://raw.githubusercontent.com/devcontainers/spec/main/schemas/devContainer.schema.json",
```

Comment:
why this?

Reply:
It raises a warning `Property $schema is not allowed.`

Reply:
that's strange, never seen that this isn't allowed. which app is complaining?

Reply:
vscode

Reply:
That's because the JSON schema for `devcontainer.json` doesn't allow the `$schema` property itself. IMO this is an oversight by upstream.

I'll create an issue upstream tomorrow (I can't interact with Microsoft owned GitHub repositories from a non-work machine)

Reply:
Opened https://github.com/devcontainers/spec/issues/259

Reply:
Opened a PR https://github.com/devcontainers/spec/pull/260

Reply:
@lmilbaum https://github.com/devcontainers/spec/pull/260 is merged upstream. Can you re-add `$schema` please?

Reply:
Thanks for taking care of it.  re-added.